release/src/main/scripts/build_release_candidate.sh

```diff
@@ -275,7 +276,7 @@ if [[ $confirmation = "y" ]]; then
   git clone ${GIT_REPO_URL}
   cd ${BEAM_ROOT_DIR}
   git checkout ${RELEASE_BRANCH}
-  cd sdks/python && tox -e docs
+  cd sdks/python && pip install -r build-requirements.txt && tox -e py37-docs
```

Comment:
Please also update the release guide:
https://github.com/apache/beam/blame/master/website/src/contribute/release-guide.md#L783

Reply:
I can do that.. but I hope we get rid of the "manual" steps in the future. They add no utility over the script and make the release guide hard to read.

Reply:
Done

Reply:
Done